CHANGELOG

```diff
@@ -1,3 +1,17 @@
+* Wed Oct 03 2018 Brandon Riden <brandon.riden@onyxpoint.com> - 1.1.0-0
+- No longer support Puppet less than 4.10.4
+- Convert README to markdown and add a REFERENCE.md
+- Removed params.pp in favor of data in modules
+- Hashes in Puppet language should have keys that are quoted strings
+- Updated acceptance test to remove unnecessary proxy check and set correct syntax
```

Comment:
proxy test was essential, as that is how we connect to ElasticSearch.  Was this test actually removed?  I thought I brought this up already.

Reply:
I'm not familiar with the history of this ticket, what was this test? I can't find it in the code.

Reply:
The proxy test **is** in the code.  Looks like the test I fixed on master got merged in.  So, this comment needs to be removed from the changelog.
Here is the test on master: https://github.com/simp/pupmod-simp-simp_grafana/blob/master/spec/acceptance/suites/default/50_grafana_spec.rb#L337 
(I added documentation so the importance of the proxy test would be clear).